source/extensions/filters/listener/http_inspector/http_inspector.cc

```diff
@@ -124,6 +125,7 @@ ParseState Filter::parseHttpHeader(absl::string_view data) {
     }
     ENVOY_LOG(trace, "http inspector: http2 connection preface found");
     protocol_ = "HTTP/2";
+    ENVOY_LOG(trace, "http inspector: set application protocol to %s", protocol_);
```

Comment:
Actually, you can log one time at https://github.com/envoyproxy/envoy/pull/20565/files#diff-461a2ff7b4e7a7715d580e27273613c95d7ef468bb6b39afcb47b7cafa1d8161R113

Then you needn't duplicated the log two times